### DISCUSSION TITLE
regexp + форматирование номера.

### DISCUSSION THREAD

**Discussion by Qb1cc** [totumonline/totum-ru-issues-and-discussions#86] [Как это сделать?] [answered]
Пользователи могут по разному вносить номера с 8 без 8 или же вообще писать +7..Нужно каким то образом форматировать полученный номер до формата 89292292929.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Ответы сейчас будут дольше, чем обычно.

Самый простой способ:

```
= : str`"8" + $str`

str: strPart(str: #field_with_number; offset: -10; length: 10)

```

Те взять последние 10 цифр и к ним проклеить 8.
